### PR TITLE
Docs: Fix Admin docs relrefs

### PR DESCRIPTION
**Pull request by gguillotte-grafana**
**What this PR does / why we need it**:

Tooling did not catch or fix all relrefs in the refactored Administration docs. This PR updates those relrefs to point to the new refactored URIs.

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:

Draft PR, ongoing updates to apply fixes.

Because tooling could not fix these automatically, these changes are implemented manually. Each link should be checked if possible to confirm that it is both valid and points to the correct destination.

### CODE REVIEW

File: docs/sources/administration/enterprise-licensing/activate-aws-marketplace-license/_index.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 aliases:
   - /docs/grafana/latest/enterprise/license/activate-aws-marketplace-license/
-description: Activate Enterprise license purchased through AWS Marketplace
   - /docs/grafana/latest/enterprise/activate-aws-marketplace-license/about-ge-license-through-aws/
   - /docs/grafana/latest/enterprise/license/activate-aws-marketplace-license/about-ge-license-through-aws/
+description: Activate Enterprise license purchased through AWS Marketplace
```
Comment (GrafanaWriter) [inline docs/sources/administration/enterprise-licensing/activate-aws-marketplace-license/_index.md:6]: nit to make this slightly more expansive:
```suggestion
description: Activate your Grafana Enterprise license purchased in AWS Marketplace to take advantage of Grafana Enterprise observability features```

**Review by chri2547 — APPROVED** [review]

**Review by GrafanaWriter — APPROVED** [review]
LGTM - tested manually - thank you

**Comment by gguillotte-grafana** [comment]
Flipping back to draft - more relref fixes coming.

**Review by GrafanaWriter — APPROVED** [review]
LGTM. Manually checked links.

**Pull request merged**